tests/antispoof/tests.c

```diff
@@ -534,14 +551,38 @@ static void test_antispoof_ndp(void)
 	pg_antispoof_ndp_enable(antispoof);
 	pg_antispoof_ndp_del_all(antispoof);
 
+	/* add NDP_MAX adresses  */
+	for (int i=0;i<=100;i++){
+		pg_ip_from_str(ip, "2001:db8:2000:aff0::"+i);
+		if(i!=100){ 
+		        pg_antispoof_ndp_add(antispoof, ip, &error);
+			g_assert(!error);
+		} else {
+			pg_antispoof_ndp_add(antispoof, ip, &error);
+                        g_assert(error);
+			pg_error_free(error);
+			error=NULL;
+		}
+	}
```

Comment:
same comment as above (except that here the block seems correct), also the indentation....